assets/collage.css

```diff
@@ -154,8 +148,15 @@
   border: 0;
 }
 
-.collage-card .deferred-media__poster:focus,
-.collage-card .collage-card__link:focus {
+.collage-card .deferred-media__poster:focus-visible,
+.collage-card .deferred-media__poster:focus {
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0.5rem 0.4rem rgba(var(--color-foreground), 0.3);
+  outline-offset: 0.3rem;
+  outline: 0.2rem solid rgba(var(--color-foreground), 0.5);
+}
+
+.collage-card .deferred-media__poster:focus:not(:focus-visible),
+.collage-card .collage-card__link:focus:not(:focus-visible) {
```

Comment:
In safari, the focus looks different for the selected video vs. the onboarding state. If you click outside of the safari window, the grey border is displayed, but not when tabbing. The focus state looks right for the regular video section.

https://user-images.githubusercontent.com/26778637/148977678-67f95c60-6cff-42e1-97aa-ab4c066adadc.mov


<img width="1580" alt="Screen Shot 2022-01-11 at 11 06 00 AM" src="https://user-images.githubusercontent.com/26778637/148978284-7c8f59f1-fece-4d66-905a-1845ea7db9eb.png">

<img width="877" alt="Screen Shot 2022-01-11 at 11 06 16 AM" src="https://user-images.githubusercontent.com/26778637/148978343-0df2dd8b-c18e-42ba-b39b-5f11c9373647.png">

Reply:
Ah I know why. I grouped selectors for `:focus` and `:focus-within` which I should not because `:focus-within` isn't supported in safari so the styling for `:focus` is not applied. I separated them and it should now be good 🙂

Reply:
Related to Safari, I noticed the focus is not following the UI shape here, I can open an issue to not busy up this PR https://screenshot.click/22-01-mtov6-l1wzh.png

Reply:
It may be that Safari is being more strict about the actual CSS spec with regards to outlines (which are not really for decorative use, technically).

Reply:
> From what I can tell most of the focus outlines/shadows, I think all of them actually, aren't receiving the border radius like in other browsers. It's specific to Safari.

Yeah. I talked to Ken and we need to convert them to box-shadow because `outline` doesn't inherit the border-radius.